CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # Changelog
-All notable changes to this project 3.0.x per each release will be documented in this file.
+All notable changes to this project 3.1.0 per each release will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## [v3.1.0.1]
+- Introduce Helm chart for WSO2 APIM version 3.1.0 deployment pattern 1
+- Introduce Helm chart for WSO2 APIM version 3.1.0 deployment pattern 2
+
 ## [v3.0.0.3] - 2020-01-06
```

Comment:
@Chathuru minor change. Please remove the change logs content for version `3.0.x` resources since the change log in `3.1.x` branch is purely for resources of that particular version.